server/utils/azureAppInsights.ts

```diff
@@ -1,5 +1,23 @@
-import { setup, defaultClient, TelemetryClient, DistributedTracingModes } from 'applicationinsights'
-import type { ApplicationInfo } from '../applicationInfo'
+import { setup, defaultClient, TelemetryClient, DistributedTracingModes, Contracts } from 'applicationinsights'
+import { EnvelopeTelemetry } from 'applicationinsights/out/Declarations/Contracts'
+import applicationVersion from '../applicationVersion'
+
+export type ContextObject = {
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  [name: string]: any
+}
+
+function defaultName(): string {
```

Comment:
this is not needed as we get this from ApplicationInfo (as per the new template)